.github/workflows/build.yml

```diff
@@ -16,38 +16,63 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v18
+        with:
+          nix_path: nixpkgs=channel:nixos-22.11
+          extra_nix_config: |
+            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
       - uses: actions/setup-go@v3
         with:
           go-version: 1.18
-      - uses: technote-space/get-diff-action@v6
-        id: git_diff
+      - id: changed-files
+        uses: tj-actions/changed-files@v34
```

Comment:
Have you found any advantage or CI speed up using this action?
We still use `get-diff-action` in the SDK, but maybe we should look into this one 👀

Reply:
We did this switch mainly to handle the github warning of using some deprecated stuff, I think "get-diff-action" is not updated at the time, so we switch to a seemingly better maintained version, which also has rich features.